api/v1alpha1/flowcollector_types.go

```diff
@@ -358,6 +358,16 @@ type FlowCollectorFLP struct {
 	// kafkaConsumerBatchSize indicates to the broker the maximum batch size, in bytes, that the consumer will accept. Ignored when not using Kafka. Default: 10MB.
 	KafkaConsumerBatchSize int `json:"kafkaConsumerBatchSize"`
 
+	//+kubebuilder:validation:Pattern:=^\d+(ns|ms|s|m)?$
+	//+kubebuilder:default:="30s"
+	// connection update interval is the duration of time to wait between update reports of a connection
+	ConnectionUpdateInterval string `json:"connectionUpdateInterval,omitempty"`
```

Comment:
If I understand correctly, we should not modify anymore the v1alpha1 CRD

Reply:
I'm not 100% sure but I believe if the field is optional, and if we consider in code that it can be empty, then it could be still done as part of v1alpha1
(cc @msherif1234 )

Reply:
Let's follow this in https://github.com/netobserv/network-observability-operator/pull/258#discussion_r1100457798

Reply:
Following the discussion in #258 v1beta1 will not be merged for now